.github/workflows/linting.yml

```
@@ -5,11 +5,21 @@ jobs:
   prose:
     runs-on: ubuntu-latest
     steps:
+    # Checkout this repo
     - name: Checkout
       uses: actions/checkout@master
```

It is safer to keep a major version pinned.
```suggestion
      uses: actions/checkout@v2
```